### PR TITLE
Add checkstyle and lint

### PR DESCRIPTION
**Pull request by sectioneight**
@jjx

### CODE REVIEW

File: config/checkstyle.xml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        "http://www.puppycrawl.com/dtds/configuration_1_2.dtd">
+
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="OneTopLevelClass"/>
+    <module name="NoLineWrap"/>
+    <module name="EmptyBlock"/>
+    <module name="Indentation"/>
+    <module name="MethodName"/>
+    <module name="EmptyLineSeparator">
```
Comment (sectioneight) [inline config/checkstyle.xml]: Note: I couldn't figure out how to check for double-newlines. Any ideas?

Comment (jjx) [inline config/checkstyle.xml]: RegexpMultiline

File: src/main/java/com/uber/jenkins/phabricator/tasks/Task.java [hunk 2]
```diff
@@ -35,10 +35,10 @@
         FAILURE,
         IGNORED, // For incorrect input.
         SKIPPED, // For incorrect configuration.
-        UNKNWON
-    };
+        UNKNOWN
```
Comment (jjx) [inline src/main/java/com/uber/jenkins/phabricator/tasks/Task.java]: LOL, thanks for fixing that

**Comment by jjx** [comment]
lgtm

File: config/checkstyle.xml (hunk 1 above)
Comment (sectioneight) [inline config/checkstyle.xml]: sweet, done (and caught a bunch of existing issues)

**Pull request merged**